api/lib/validation/schemaValidator.js

```diff
@@ -57,7 +57,7 @@ var getDaysInMonthInISOFormat = function() {
 };
 
 var getMetricTypes = function() {
-  var metricTypeEnum = ['memoryused'];
+  var metricTypeEnum = ['memoryutil', 'responsetime', 'throughput'];
```

Comment:
@boyang9527 
Correct me if I'm wrong.
We're not getting quota while using polling metrics, quota is only available while streaming metrics so I think we should keep "memoryused" too and apply the upper limit validation selectively.